R/iNZChangeDataWin.R

```diff
@@ -1022,12 +1022,9 @@ iNZjoinDataWin <- setRefClass(
         string1 <- glabel("Preview of the original dataset")
         originview <- gtable(data.frame(head(GUI$getActiveData(), 10)))
         string2 <- glabel("Select join methods")
-        var1 <- gcombobox(
-	    items = c("Inner Join", "Left Join", "Full Join", "Semi Join", "Anti Join"),
-	    selected = 2
-        )
-	join_method <<- svalue(var1)
-
+        var1 <- gcombobox(items = c("Inner Join", "Left Join", "Full Join", "Semi Join", "Anti Join"), selected = 2)
```

Comment:
This should be OK for now. Should probably create a mapping table to make it easier to change this in future.
```r
jointypes <- list("Inner Join" = "inner_join", "Left Join" = "left_join", ...)
var1 <- gcombobox(items = names(jointypes), selected = 2) # this is all you now need to change to adjust default
join_method <- jointypes[[svalue(var1)]]
```

Reply:
Your switch later down could then use the jointypes also.